packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -0,0 +1,544 @@
+
+  /**
+   * The VPC that this Aurora Serverless cluster has been created in.
+   *
+   */
+  readonly vpc: ec2.IVpc;
+
+  /**
+   * Where to place the instances within the VPC
+   *
+   * @default - the VPC default strategy if not specified.
+   */
+  readonly vpcSubnets?: ec2.SubnetSelection;
+
+  /**
+   * Scaling configuration
+   *
+   * @default - None
+   */
+  readonly scaling?: ServerlessScalingOptions;
+
+  /**
+   * The removal policy to apply when the cluster and its instances are removed
+   * from the stack or replaced during an update.
+   *
+   * @default - RemovalPolicy.SNAPSHOT (remove the cluster and instances, but retain a snapshot of the data)
+   */
+  readonly removalPolicy?: RemovalPolicy;
+
+  /**
+   * Security group.
+   *
+   * @default - a new security group is created.
+   */
+  readonly securityGroups?: ec2.ISecurityGroup[];
+
+  /**
+   * The KMS key for storage encryption.
+   *
+   * @default - the default master key will be used for storage encryption
+   */
+  readonly storageEncryptionKey?: kms.IKey;
+
+  /**
+   * Additional parameters to pass to the database engine
+   *
+   * @default - no parameter group.
+   */
+  readonly parameterGroup?: IParameterGroup;
+}
+
+/**
+ * Create a clustered database with a given number of instances.
+ */
+export interface IServerlessCluster extends IResource, ec2.IConnectable, secretsmanager.ISecretAttachmentTarget {
+  /**
+   * Identifier of the cluster
+   */
+  readonly clusterIdentifier: string;
+
+  /**
+   * The endpoint to use for read/write operations
+   * @attribute EndpointAddress,EndpointPort
+   */
+  readonly clusterEndpoint: Endpoint;
+
+  /**
+   * Endpoint to use for load-balanced read-only operations.
+   * @attribute ReadEndpointAddress
+   */
+  readonly clusterReadEndpoint: Endpoint;
+}
+
+/**
+ * Properties that describe an existing cluster instance
+ */
+export interface ServerlessClusterAttributes {
+  /**
+   * Identifier for the cluster
+   */
+  readonly clusterIdentifier: string;
+
+  /**
+   * The database port
+   *
+   * @default - none
+   */
+  readonly port?: number;
+
+  /**
+   * The security groups of the database cluster
+   *
+   * @default - no security groups
+   */
+  readonly securityGroups?: ec2.ISecurityGroup[];
+
+  /**
+   * Cluster endpoint address
+   *
+   * @default - no endpoint address
+   */
+  readonly clusterEndpointAddress?: string;
+
+  /**
+   * Reader endpoint address
+   *
+   * @default - no reader address
+   */
+  readonly readerEndpointAddress?: string;
+}
+
+/**
+ * Aurora capacity units (ACUs).
+ * Each ACU is a combination of processing and memory capacity.
+ *
+ * @see https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless.setting-capacity.html
+ * @see https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless.how-it-works.html#aurora-serverless.architecture
+ */
+export enum AuroraCapacityUnit {
+  /**
+   * 1 Aurora Capacity Unit
+   */
+  ACU_1 = 1,
```

Comment:
`~` Nit - Can you inline these a bit more, just to use up less space?
```suggestion
  /** 1 Aurora Capacity Unit */
  ACU_1 = 1,
```

Reply:
done